clientlibrary/checkpoint/dynamodb-leasestealer.go

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (c) 2018 VMware, Inc.
```

Comment:
Nit. 2020

Reply:
Resolving. This file is no longer a part of the PR